src/main/scala/com/fortysevendeg/scalacheck/datetime/instances/j8.scala

```diff
@@ -0,0 +1,15 @@
+package com.fortysevendeg.scalacheck.datetime.instances
+
+import com.fortysevendeg.scalacheck.datetime.typeclasses._
+import java.time._
+import java.time.temporal.ChronoUnit.MILLIS
+
+trait J8Instances {
+
+  implicit val j8ForDuration: ScalaCheckDateTimeInfra[ZonedDateTime, Duration] = new ScalaCheckDateTimeInfra[ZonedDateTime, Duration] {
```

Comment:
Would it be a good idea to replace it for `implicit object j8ForDuration extends ScalaCheckDateTimeInfre[ZonedDateTime, Duration] `?
I find my self doing a lot of those these days, although I do not know if they are better than `vals`.